File: app/src/main/java/com/amarnehsoft/holyquran/main/readersFragment/ReaderController.java
package com.amarnehsoft.holyquran.main.readersFragment;

import com.amarnehsoft.holyquran.test.Reader;

public class ReaderController {
    private static Reader currentReader = Reader.mahermuaiqly;

    public static void changeReader(Reader readers){
        currentReader = readers;
    }

    public static Reader getCurrentReader(){
        return currentReader;
    }
}

File: app/src/main/java/com/amarnehsoft/holyquran/main/readersFragment/ReaderController.kt
package com.amarnehsoft.holyquran.main.readersFragment

import com.amarnehsoft.holyquran.test.Reader

object ReaderController {
    var currentReader = Reader.MaherMuaiqly
        private set

    fun changeReader(readers: Reader) {
        currentReader = readers
    }
}
